src/main.rs

```diff
@@ -27,7 +27,8 @@ fn show_help() {
     println!("If no config file is given. The file routes/default will be used.");
 }
 
-fn main() -> Result<(), Box<dyn Error>> {
+fn main() -> Result<(), color_eyre::Report> {
```

Comment:
The [`color_eyre::Report`](https://docs.rs/eyre/latest/eyre/struct.Report.html) type enables custom output of errors